Settings.h

```diff
@@ -70,6 +73,14 @@ typedef struct Settings_ {
 
 #define Settings_cpuId(settings, cpu) ((settings)->countCPUsFromOne ? (cpu)+1 : (cpu))
 
+#define Settings_getActiveSortKey(settings) (((settings)->treeView) \
+                      ? ((settings)->treeViewAlwaysByPID ? PID : (settings)->treeSortKey) \
+                      : ((settings)->sortKey))
+
+#define Settings_getActiveDirection(settings) (((settings)->treeView) \
+                      ? ((settings)->treeDirection) \
+                      : ((settings)->direction))
+
```

Comment:
Please prefer (`static inline`) function implementation over inline macros.

Reply:
And place them right here in the .h file?

Reply:
Done!